build-tools/build.sh

```diff
@@ -77,8 +77,10 @@ build_registry() {
     if [[ -f "stack.yaml" ]]; then
       for versionDir in $stackDir/*
       do
-        cd $versionDir
-        tar_files_and_cleanup
+        if [[ -d "${versionDir}" ]]; then
+          cd $versionDir
+          tar_files_and_cleanup
+        fi
```

Comment:
Do we need to "cd" back to the parent dir if there are multiple version dirs?

Reply:
@kim-tsao Not in this case, since `$versionDir` is absolute it does not matter.